coops-ui/src/resources/business-config.ts

```diff
@@ -0,0 +1,66 @@
+export class Flow {
+  feeCode: string
+  displayName: string
+  certifyText: string
+}
+
+export class Business {
+     typeEnum: string
+     displayName: string
+     flows: Array<Flow>
+}
+
+export const configJson = [{
+  typeEnum: 'BC',
+  displayName: 'Benefits Company',
+  flows: [
+    {
+      feeCode: 'OTADD',
+      displayName: 'Change Of Address',
+      certifyText: 'Note: It is an offence to make a false or misleading statement in' +
+           'respect of a material fact in a record submitted to the Corporate Registry for filing. ' +
+           'See sections 35 and 36 of the Business Corporations Act.'
+    },
+    {
+      feeCode: 'OTANN',
+      displayName: 'Annual Report',
+      certifyText: 'Note: It is an offence to make a false or misleading statement in' +
+           'respect of a material fact in a record submitted to the Corporate Registry for filing. ' +
+           'See section 51 of the Business Corporations Act.'
+    },
+    {
+      feeCode: 'OTCDR',
+      displayName: 'Change Of Directors',
+      certifyText: 'Note: It is an offence to make a false or misleading statement in' +
+           'respect of a material fact in a record submitted to the Corporate Registry for filing. ' +
+           'See section 127 of the Business Corporations Act.'
+    }
+  ]
+},
+{
+  typeEnum: 'CP',
```

Comment:
Same as above, i think we have an enum for entity types. Does it make sense to them in the config files?

Reply:
similar to my other comment :)